src/kOS/Suffixed/AggregateResourceValue.cs

```diff
@@ -44,9 +45,8 @@ public override string ToString()
             return string.Format("SHIPRESOURCE({0},{1},{2})", name, amount, capacity);
         }
 
-        public static ListValue PartsToList(IEnumerable<global::Part> parts, SharedObjects shared)
```

Comment:
this class had two methods that did almost exactly the same task but had different return types. this unifies them for DRY